<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupWidget</class>
 <widget class="QWidget" name="BackupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="trashButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>291</width>
     <height>63</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="BackupName">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NomSauvegarde</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="BackupSource">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Source</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="BackupCible">
      <property name="text">
       <string>Cible</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
